pkg/util/validation/errors.go

```diff
@@ -275,15 +275,15 @@ func NewMaxQueryLengthError(actualQueryLen, maxQueryLength time.Duration) LimitE
 		maxQueryLengthFlag))
 }
 
-func NewRequestRateLimitedError(limit float64, burst int) LimitError {
+func NewRequestRateLimitedError(globalLimit, limit float64, burst int) LimitError {
 	return LimitError(globalerror.RequestRateLimited.MessageWithLimitConfig(
-		fmt.Sprintf("the request has been rejected because the tenant exceeded the request rate limit, set to %v req/s with a maximum allowed burst of %d", limit, burst),
+		fmt.Sprintf("the request has been rejected because the tenant exceeded the request rate limit, set to %v requests/s across all distributors and %v requests/s per distributor with a maximum allowed burst of %d", globalLimit, limit, burst),
```

Comment:
I would remove the part "and %v requests/s per distributor". I think it just creates confusion to the user, without adding much value.

_Same comment applies below._